Change inv method to adjoint method in unit tests (#88)
PennyLaneAI/pennylane-lightning-gpu: .github/CHANGELOG.md

### Improvements

> * Change `inv()` method to `qml.adjoint()` method in Python unit tests to be aligned with recent change in Pennylane and avoid CI failures.

Reviewer: ```suggestion
* Update `inv()` to `qml.adjoint()` in Python tests following recent changes in Pennylane.
```

Author: Thanks @AmintorDusko ! Let me update it.
